src/components/App.vue

```diff
@@ -44,36 +50,37 @@ export default {
   }
 
   $edge-board-size: 100px;
+  $admin-board-size: constants.$reset-button-size;
   #root {
     display: grid;
-    grid-template-columns: $edge-board-size auto;
-    grid-template-rows: $edge-board-size auto $edge-board-size;
+    grid-template:
```

Comment:
IMO it's best to separate restructuring changes from functional changes to make PRs easier to understand for the reviewer. I would have put the `grid-template` restructure in one commit and the new admin column in another commit.

Reply:
Yup, I know I gotta work on more atomic commits